src/components/ChatEntry.js

```diff
@@ -1,22 +1,50 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const id = props.id;
+  const sender = props.sender;
+  const body = props.body;
+  const timeStamp = props.timeStamp;
+  const liked = props.liked;
+  const updateLiked = props.updateLiked;
+  const display = sender === 'Vladimir' ? 'local' : 'remote';
```

Comment:
This works for the data in the json file, but how could you make this more general so it works for any name of the sender?